Included more instructions to build the linux executable using docker. (#6)
ofabricio/capture: README.md

Now you have an executable binary in your directory.

> **Note:** change `GOOS=darwin` to `linux` or `windows` to create an executable for your corresponding Operating System. For linux, is also needed to change the image tag as well: `golang:1.16`.

Reviewer: ```suggestion
**Note:** change `GOOS=darwin` to `linux` or `windows` to create an executable for your corresponding Operating System. For linux, you also need to change the image tag to `golang:1.16`.
```
